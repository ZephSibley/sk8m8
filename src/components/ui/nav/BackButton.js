import React from 'react';
import ArrowBackIcon from '@material-ui/icons/ArrowBack';
import { makeStyles } from '@material-ui/core/styles';

const BackButton = () => {

    const useStyles = makeStyles({
        root: {
            //width: '100vw',
            position: 'fixed',
            top: 10,
            left: 5,
        },
    });

    const classes = useStyles();
    return (
        <ArrowBackIcon
            className={classes.root}
            onClick={() => window.history.back()}
        />
    )
}

export default BackButton;