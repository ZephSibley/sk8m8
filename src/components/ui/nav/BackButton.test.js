import React from 'react';
import renderer from 'react-test-renderer';
import BackButton from './BackButton';

test('Basic Rendering', () => {
    const component = renderer.create(
        <BackButton />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot();
})