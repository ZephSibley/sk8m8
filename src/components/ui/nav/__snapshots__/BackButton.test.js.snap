// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Rendering 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root makeStyles-root-1"
  focusable="false"
  onClick={[Function]}
  role="presentation"
  viewBox="0 0 24 24"
>
  <path
    d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
  />
</svg>
`;
